sample_size/scripts/sample_size_run.py

```diff
@@ -15,19 +15,32 @@ def main() -> None:
     from sample_size.sample_size_calculator import SampleSizeCalculator
     from sample_size.scripts.input_utils import get_alpha
     from sample_size.scripts.input_utils import get_metric_metadata
+    from sample_size.scripts.input_utils import get_mde
+    from sample_size.scripts.input_utils import get_variants
+    from sample_size.scripts.input_utils import register_another_metric
 
     try:
         # Get alpha for power analysis
         alpha = get_alpha()
-        calculator = SampleSizeCalculator(alpha)
+        variants = get_variants()
+        calculator = SampleSizeCalculator(alpha, variants)
 
-        # register metric
-        metric_type, metric_metadata = get_metric_metadata()
-        calculator.register_metric(metric_type, metric_metadata)
+        register = True
+        while register:
```

Comment:
can we move the codes for registration to `get_metric_metadata` under `input_utils.py`?
here can just be 
```
alpha = get_alpha()
variants = get_variants()
calculator = SampleSizeCalculator(alpha, variants)

metric_types, metric_metadatas = get_metric_metadata()
calculator.register_metric(metric_types, metric_metadatas)

# Get and print sample size based on variable and power analysis parameters
sample_size = calculator.get_sample_size()
```

Reply:
The current calculator gives me this error
<img width="808" alt="image" src="https://user-images.githubusercontent.com/97130511/167035536-83172043-5666-4ad5-a8b2-730553374460.png">